client/src/actions/authActions.js

```diff
@@ -2,8 +2,8 @@ import axios from "axios";
 import { Toast } from "antd-mobile";
 
 import { GET_ERRORS } from "./types";
-import { AUTH_LOGIN, AUTH_SIGNUP } from "constants/action-types";
-import { setAuthToken, getAuthToken } from "utils/auth-token";
```

Comment:
Yeah so this wasn't added because the local storage was a bit of a temporary thing when first developing, currently it wasn't really specified in the requirements and I think out of scope here to store the JWT in local storage after every login.

Reply:
Lines commented.